dev-python/js2py/files/js2py-0.74-CVE-2024-28397.patch

```diff
@@ -0,0 +1,21 @@
+# https://nvd.nist.gov/vuln/detail/CVE-2024-28397
```

Comment:
bump + patch? is this not on js2py master branch? - if so, lest do a snapshot of it's master